README.md

```diff
@@ -8,7 +8,7 @@ This repository builds jellifin-tizen with three commands. The resulting file is
 
 ## Build process:
 
-    DOCKER_BUILDKIT=0 docker build --ulimit nofile=65536:65536 -t jellyfin-tizen-build .
+    docker build --ulimit nofile=65536:65536 --progress=plain --no-cache -t jellyfin-tizen-build .
```

Comment:
(Minor) Option `--progress=plain` is just for full build log, I don't think it's necessary

Reply:
I personally prefer to have a full log in my terminal scrollback buffer, so I like this option. I'll leave it up to @gardner to remove it again :smiley: